tests/integration-service/integration.go

```diff
@@ -263,6 +265,69 @@ var _ = framework.IntegrationServiceSuiteDescribe("Integration Service E2E tests
 			Expect(f.AsKubeAdmin.CommonController.HaveTestsSucceeded(snapshot)).To(BeFalse(), "expected tests to fail for snapshot %s/%s", snapshot.GetNamespace(), snapshot.GetName())
 		})
 
+		It("creates a new IntegrationTestScenario", FlakeAttempts(3), func() {
```

Comment:
out of curiosity, why so many test specs added in this PR have `FlakeAttempts(3)` defined?

Reply:
I may have went a little overboard with checks that might be flaky. This is not an instance that should have that issue, removed it and left it only on places where it makes sense.